tests/unit/style/properties/serialization.rs

```diff
@@ -1231,4 +1231,26 @@ mod shorthand_serialization {
             assert_eq!(serialization, block_text);
         }
     }
+      mod effects {
+        pub use super::*;
+        pub use style::properties::longhands::box_shadow::SpecifiedValue as BoxShadow;
+        pub use style::values::specified::Shadow;
+        #[test]
+        fn box_shadow_should_serialize_correctly() {
+            let mut properties = Vec::new();
+            let color = Some(CSSColor {
```

Comment:
You can just move it to the top of the file making it `pub` (above the first `mod`), and use it from the others.